translations/messages.en.po

```diff
@@ -27830,6 +27830,21 @@ msgstr "Autolaunch settings"
 msgid "Auto-launch"
 msgstr "Auto-launch"
 
+msgid "Link classes to created groups ?"
+msgstr "Link classes to created groups ?"
+
+msgid "Info message about deactivation of consistent link"
+msgstr "If desactivated, the classes linked to this group are simply a copy of the users of the selected class. But, if it is enabled, the selected class is linked to the group, and modifications to the class are reflected in the group (this mode prevents modification of the group)."
+
+msgid "Warning message to warn that the user cannot modify members of linked group"
+msgstr "This group is linked to a class. Any additions or deletions of members to the class will be push the group. You cannot add or remove members to a group linked to a class. However you can break this link by clicking on the button below"
+
+msgid "Remove the group link with the class"
+msgstr "Remove the group link with the class"
+
+msgid "Group is no longer linked to the course"
+msgstr "Group is no longer linked to the course"
+
```

Comment:
This has the wrong text msgid for those messages like "Info message about ..." and "Warning message to warn..." (should be updated together with the code).
It is also missing the various strings that are currently in French in the code.

Also, try to always add your new strings *at the end* of the file. No matter the group of strings or any alphabetical order.